### PR TITLE
made my profile for Linkfree

### PR DESCRIPTION
**Pull request by itsomsarraf**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1509"></a>

### CODE REVIEW

File: public/data/itsomsarraf_.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/itsomsarraf_",
+        "icon": "twitter"
+      },
+      {
+        "name": "Connect with me on Linkedin",
+        "url": "https://www.linkedin.com/in/itsomsarraf/?_l=en_US",
```
Comment (kumarsonsoff3) [inline public/data/itsomsarraf_.json:19]:
```suggestion
        "url": "https://www.linkedin.com/in/itsomsarraf/",
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: public/data/itsomsarraf_.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        "url": "https://github.com/itsomsarraf",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/itsomsarraf_",
+        "icon": "twitter"
+      },
+      {
+        "name": "Connect with me on Linkedin",
```
Comment (Panquesito7) [inline public/data/itsomsarraf_.json:18]: What do you think about capitalizing `i` here?
```suggestion
        "name": "Connect with me on LinkedIn",
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing!


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Maintainable

**Review by jaynarayan-vaishnav — APPROVED** [review]
LGTM🚀

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by kumarsonsoff3 — APPROVED** [review]

**Pull request merged**